packages/components/src/components/Select/Select.tsx

```diff
@@ -0,0 +1,136 @@
+/* (c) Copyright Frontify Ltd., all rights reserved. */
+
+import { IconCaretDown } from '@frontify/fondue-icons';
+import * as RadixPopover from '@radix-ui/react-popover';
+import { Slot as RadixSlot } from '@radix-ui/react-slot';
+import { useSelect } from 'downshift';
+import { forwardRef, type ForwardedRef, type ReactNode } from 'react';
+
+import { ForwardedRefCombobox } from './Combobox';
+import { ForwardedRefSelectItem, ForwardedRefSelectItemGroup } from './SelectItem';
+import { SelectMenu } from './SelectMenu';
+import { ForwardedRefSelectSlot } from './SelectSlot';
+import styles from './styles/select.module.scss';
+import { useSelectData, type SelectItemType } from './useSelectData';
+
+export type SelectComponentProps = {
+    /**
+     * Children of the Select component. This can contain the `Select.Slot` components for the label, decorators, clear action and menu.
+     */
+    children?: ReactNode;
+    /**
+     * Callback function that is called when an item is selected.
+     */
+    onSelect?: (selectedItem: SelectItemType) => void;
+    /**
+     * The active item in the select component. This is used to control the select externally.
+     */
+    activeItem?: SelectItemType;
+    /**
+     * The default value of the select component. Used for uncontrolled usages.
+     */
+    defaultValue?: string;
+    /**
+     * The placeholder in the select component.
+     */
+    placeholder?: string;
+    /**
+     * Disables the select component.
+     */
+    disabled?: boolean;
+    /**
+     * The aria label of the select component.
+     */
+    'aria-label': string;
```

Comment:
Let's align it with the one in Combobox?

Reply:
changed